File: R/ex5.R
#!/bin/Rscript

## Exercício 2.9
main <- function()
{
  cat("Informe um n: ")
  n <- as.numeric(readLines("stdin", n = 1))
  
  vetor <- c()
  
  for(i in 1:n)
  {
    vetor <- c(vetor, 1:i)
  }
  
  return(vetor)
}

main()

File: py/ex5.py
#!/bin/python

## Exercício 2.9
def main():
  n = int(input("Informe um n: "))

  seq_n(n)


def seq_n(n):
  lista = []

  for i in range(1, n+2):
    lista = [f' {x} '.format(x) for x in map(str, list(range(1,i)))]
    print("".join(lista))


if __name__ == "__main__":
  main()
